contents/newsletter/talk-to-users.md

```diff
@@ -0,0 +1,106 @@
+---
+title: An engineer's guide to talking to users
+date: 2024-04-18
+author:
+  - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Engineering
+  - Product engineers
+crosspost:
+  - Product engineers
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, user researchers, and salespeople? Why do would an engineer ever want or need to do it?
+
+In his book [*Modern Software Engineering*](https://www.davefarley.net/?p=352), Dave Farley writes that mastering software engineering requires becoming an expert in learning. When learning is prioritized, talking to users becomes a critical part of an engineer's toolkit.
+
+To explain further why this is the case, and to help you get started, we wrote this guide based on what we've learned talking to users as engineers ourselves at PostHog.
+
+## Why should engineers talk to users?
+
+Engineers' biggest pushback against talking to users is that it is a waste of time. This belief is a symptom of the wrong priorities.
+
+Many engineers place shipping code above all else. Anything that gets in thwe way of this is a waste of time. This is reflected in common metrics measuring engineering success like lead time or deployment frequency. 
+
+This creates a disregard for shipping the *right* code: features users value. Studies show this is more common than you might think. For example, [Microsoft](https://ai.stanford.edu/~ronnyk/ExPThinkWeek2009Public.pdf) found that **only one-third** of ideas tested improved metrics they were designed for.
+
+A fix for this is spending more time learning what users want. Nothing beats doing this yourself directly. By doing this, you gain the knowledge to that closes the gap between what users value and what you are building.
+
+![Don't build the wrong features](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470851/posthog.com/contents/images/newsletter/talk-to-users/features.jpg)
+
+## Finding users to talk with
+
+Just because we convinced you that talking to users isn't a waste of time does not mean we want to waste your time doing so. We realize engineers are busy, but that isn't an excuse to not talk to users. As Farley writes:
+
+> Modern teams fight with schedule pressure, quality, and maintainability of their designs. They often struggle to identify the ideas that really land with users, and they fail to allow themselves the time to learn about the problem domain, the technology, and the opportunities to get something great into production.
+
+The lucky for you, you don't need a lot of users. This doesn't require a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
```

Comment:
```suggestion
Lucky for you, you don't need a lot of users. This doesn't require a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
```

Reply:
what does the 'this' in 'This doesn't require a statistically significant...' refer to?